keylime/cloud_verifier_tornado.py

```diff
@@ -44,6 +45,12 @@
     logger.error("Error creating SQL engine or session: %s", err)
     sys.exit(1)
 
+try:
```

Comment:
It maybe useful to log an info here to inform users a durable attestation import has been loaded.

Reply:
yes